params/config.go

```diff
@@ -461,6 +461,7 @@ type ChainConfig struct {
 
 	// Puffy hardfork fix the wrong order in system transactions included in a block
 	PuffyBlock *big.Int `json:"puffyBlock,omitempty"` // Puffy switch block (nil = no fork, 0 = already on activated)
+	BubaBlock  *big.Int `json:"bubaBlock,omitempty"`  // Buba switch block (nil = no fork, 0 = already on activated)
```

Comment:
is this mandatory? any dangerous if I don't specify this param in my configuration? (Ops perspective)

Reply:
I believe that if this param is not provided in json input, this field is set to `nil` when unmarshalling, if this field is `nil`, it means that `isBuba` always returns false